<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ioWindowRigWizard</class>
 <widget class="QMainWindow" name="ioWindowRigWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IBL Rig Wizard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="uiFrameAlyx">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="2">
        <widget class="QGroupBox" name="uiGroupProcedures">
         <property name="title">
          <string>Procedures</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QListView" name="uiListProcedures">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QGroupBox" name="uiGroupProjects">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Projects</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QListView" name="uiListProjects">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="uiGroupSubject">
         <property name="title">
          <string>Subject</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="uiComboSubject">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QGroupBox" name="uiGroupFlow">
         <property name="title">
          <string>Flow</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="uiPushFlush">
            <property name="text">
             <string>Flush</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uiPushStart">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="uiGroupTask">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Task</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="uiComboTask">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>26</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="uiGroupUser">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Alyx user</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="uiComboUser"/>
          </item>
          <item>
           <widget class="QPushButton" name="uiPushConnect">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="uiMenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="uiMenuTemplates">
     <property name="title">
      <string>templates</string>
     </property>
     <addaction name="uiActionTrainingTemplate"/>
     <addaction name="uiActionEphysTemplate"/>
    </widget>
    <addaction name="uiActionLoadSession"/>
    <addaction name="separator"/>
    <addaction name="uiActionRecent"/>
    <addaction name="separator"/>
    <addaction name="uiMenuTemplates"/>
   </widget>
   <addaction name="uiMenuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="uiActionLoadSession">
   <property name="text">
    <string>load session</string>
   </property>
  </action>
  <action name="uiActionRecent">
   <property name="text">
    <string>recent</string>
   </property>
  </action>
  <action name="uiActionTrainingTemplate">
   <property name="text">
    <string>training rig</string>
   </property>
  </action>
  <action name="uiActionEphysTemplate">
   <property name="text">
    <string>ephys rig</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
